src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -4547,6 +4552,13 @@ private SymbolInfo GetSymbolInfoFromNode(SyntaxNode node, CancellationToken canc
                     return this.GetSymbolInfo(cref, cancellationToken);
                 case SelectOrGroupClauseSyntax selectOrGroupClause:
                     return this.GetSymbolInfo(selectOrGroupClause, cancellationToken);
+                case OrderByClauseSyntax orderByClauseSyntax when orderByClauseSyntax.Orderings[0].AscendingOrDescendingKeyword.Kind() == SyntaxKind.None:
```

Comment:
❓ Are we sure `orderByClauseSyntax.Orderings[0]` is never null? I'm particularly interested in incomplete code like this:

```csharp
_ = from x in new[] { 0 }
    where x >= 0
    orderby
```

Reply:
Test added in commit 648dbae. This seems to be fine.